gwcs/tests/test_wcs.py

```diff
@@ -168,7 +168,7 @@ def test_from_fiducial_composite():
     celestial = cf.CelestialFrame(reference_frame=sky.frame, unit=(sky.spherical.lon.unit,
                                   sky.spherical.lat.unit), axes_order=(1, 2))
     coord_frame = cf.CompositeFrame([spec, celestial], name='cube_frame')
-    w = wcs_from_fiducial([.5 * u.micron, sky], coord_frame, projection=tan)
+    w = wcs_from_fiducial([.5, sky], coord_frame, projection=tan)
```

Comment:
This test is skipped and shouldn't be failing. I intended to keep it like this as a reminder to fix it or decide how it should be handled. But I'll create an issue now.

Reply:
oh well in that case it works now :laughing: if you create the model with units, and call it with units it's ok!